### ISSUE TITLE
Any life here?

### ISSUE THREAD

**Issue by winnieXY** [elbowz/ha-homie#16]
Hey,

is there any life here? the homie integration is nice, but it's currently broken with homeassistant. Would be nice to get it back in shape.

**Comment by elbowz** [comment]
You're right, but I've already tried to keep ha-homie compatible with HA, but it seems to introduce many breaking changes with every new release...and I don't have the time to follow this issue.

Sorry about that.

Of course, any PRs are always welcome
